lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -484,12 +484,15 @@ public function getAlterTableSQL(TableDiff $diff)
             }
 
             $columnDef    = $column->toArray();
-            $queryParts[] = 'ADD ' . $this->getColumnDeclarationSQL($column->getQuotedName($this), $columnDef);
-
+            $addColumnSql = 'ADD ' . $this->getColumnDeclarationSQL($column->getQuotedName($this), $columnDef);
             if (isset($columnDef['default'])) {
```

Comment:
This is out of the scope of this issue but it looks like this code incorrectly handles the `DEFAULT NULL` case.